### ISSUE TITLE
Collect information on who is camping with whom

### ISSUE THREAD

**Issue by kitsuta** [magfest/ubersystem#1062]
From the Magstock wishlist doc:

>need a way to collect who people are coming with / camping with (i.e. I’m camping with Joe/Jane/etc).  this is an asap thing.  The idea that I had to tackle this was to basically modify the group field from main mag’s uber.  Have there be a designated “Tent Leader”.  I was thinking it could be something like a field of  “Who is your Tent Leader? (The person coordinating your tent/cabin)”
>* have a space for a name entry and an option for self
>* if self is selected expand to have name entries for people who will be camping in their tent/cabin. I’d say 12 spaces would be more than enough

**Comment by kitsuta** [comment]
Hm, right now this seems to be implemented as just a collection field. Do we care about improving it for this year?

**Comment by EliAndrewC** [comment]
I proposed the following in a reply in the original email chain:

> I can definitely implement something more complicated than the simple text box that's there now.  How does this sound: a dropdown exists which asks them to select whether they are a Team Leader.  If they select "no" it asks them for the name of their team leader.  If they select "yes" then it asks them for the names of everyone they're camping with.

The feedback I got was that this is fine.  So basically, it'll be:
* a dropdown for whether or not you're the Team Leader
* If you are, then a textarea box appears where people can enter the names of the people in their Team

For now it'll probably just be a text box rather than anything fancy.  (We could do autocompleting of names, but that would effectively mean that we're publishing a list of everyone who's preregistered for MAGStock, which has privacy implications.)

I can get started on this and hopefully finish it tonight after work.
